include/pmacc/dimensions/DataSpace.hpp

```diff
@@ -57,17 +57,14 @@ namespace pmacc
         static constexpr uint32_t Dim = T_dim;
         using BaseType = math::Vector<int, T_dim>;
 
-        /**
-         * default constructor.
+        /** default constructor.
+         *
          * Sets size of all dimensions to 0.
          */
-        HDINLINE DataSpace()
+        HDINLINE constexpr DataSpace() : math::Vector<int, T_dim>(math::Vector<int, T_dim>::create(0))
```

Comment:
```suggestion
        HDINLINE constexpr DataSpace() : BaseType(BaseType::create(0))
```
Not very important, but possibly cleaner